axevent/subscribe_to_event/app/subscribe_to_event.c

```diff
@@ -111,7 +111,7 @@ int main(void) {
     GMainLoop* main_loop          = NULL;
     AXEventHandler* event_handler = NULL;
     guint subscription            = 0;
-    guint token                   = 1234;
+    gdouble token                 = 1234;
```

Comment:
What's the reason to change the token to a double?
There are other parts of the code where the token is still using guint, e.g. subscription_callback().